lib/entities/asset.js

```diff
@@ -117,14 +117,14 @@ const ASSET_PROCESSING_CHECK_RETRIES = 5
  * @property {Asset[]} items - an array of assets
  * @property {function(): Object} toPlainObject - Returns this Asset collection as a plain JS object
  * @example
- * 
+ *
  * // require contentful-management
  * var contentfulManagement = require('contentful-management')
  * var client = contentfulManagement.createClient({
  * // This is the access token for this space. Normally you get both ID and the token in the Contentful web app
  * accessToken: 'YOUR_ACCESS_TOKEN'
- * }) 
- * 
+ * })
+ *
  * let space = await client.getSpace('SPACE_ID')
```

Comment:
Not really part of this PR but I feel using `await` is a bit premature since most people won't be able to run it just like that. I think we shouldn't use it in examples until it's widely available.